r/NAMESPACE

```diff
@@ -178,5 +177,6 @@ importFrom(rlang,dots_n)
 importFrom(rlang,list2)
 importFrom(rlang,warn)
 importFrom(tibble,as_tibble)
+importFrom(withr,defer)
```

Comment:
I think I'm going to let go of the `withr` dependency altogether. We don't use it much, this can be replaced by usual `on.exit()` stuff.

Reply:
I removed dependency on with here:  https://github.com/apache/arrow/pull/4025

Reply:
K. The merge conflict should not be bad if we merge this and then remove `defer`